### PR TITLE
Issue #14859: Make the jwt-1.0 feature Jakarta EE9 capable.

### PR DESCRIPTION
**Pull request by jvanhill**

### CODE REVIEW

**Comment by jvanhill** [comment]
@jhanders34 - Please review the feature files.
@aguibert  - Please review the changes to the test infrastructure. I added the ability to nest repeats and the ability to add custom locations for server configuration files.

**Comment by jvanhill** [comment]
#build
#libby

**Comment by jvanhill** [comment]
#build
#libby

**Comment by jvanhill** [comment]
#build
#libby

**Review by ryanesch — APPROVED** [review]

**Review by aguibert — APPROVED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature [hunk 1]
```diff
@@ -5,15 +5,17 @@ visibility=public
...
+-features=\
+  com.ibm.wsspi.appserver.webBundle-1.0, \
   com.ibm.wsspi.appserver.webBundleSecurity-1.0, \
-  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0"
--jars=com.ibm.websphere.appserver.api.jwt; location:=dev/api/ibm/
--files=dev/api/ibm/javadoc/com.ibm.websphere.appserver.api.jwt_1.1-javadoc.zip
--bundles=com.ibm.ws.security.jwt, \
+  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0, 5.0"
+-jars=\
+  com.ibm.websphere.appserver.api.jwt; location:=dev/api/ibm/
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature:13]: com.ibm.websphere.appserver.api.jwt will need to move to the javaee auto feature and the com.ibm.websphere.appserver.api.jwt/bnd.bnd file will need to be updated to do jakartaeeMe and a new API bundle created and reference in the jakarta auto feature.

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature]: Made the requested changes.

**Review by jhanders34 — APPROVED** [review]

**Comment by jvanhill** [comment]
#build
#libby

**Comment by jvanhill** [comment]
The builds had issues due to disk space. Re-spinning.

#build

**Pull request merged**